scripts/setup-database.sh

```diff
@@ -79,6 +81,21 @@ for db in $(echo ${POSTGRES_DBNAME} | tr ',' ' '); do
         fi
 done
 
+# Create schemas in the DB
+for db in $(echo ${POSTGRES_DBNAME} | tr ',' ' '); do
+    for schemas in $(echo ${SCHEMA_NAME} | tr ',' ' '); do
```

Comment:
Shouldn't we use `schema` (singular) instead of `schemas`. Just to be clear in the SQL statement.

Should we also deal with SQL injection too in here? :sweat_smile: 
But I don't think it's that important since the one who are able to provide these commands are the db admin.